.github/ISSUE_TEMPLATE/new-dataset-submission.md

```diff
@@ -0,0 +1,59 @@
+---
+name: New dataset submission
+about: "Submit a new dataset to mlcast-datasets by opening an issue. We’ll guide you
+  through the process \U0001F680\U0001F30D "
+title: Adding the [dataset-name] dataset
+labels: new dataset
+assignees: ''
+
+---
+
+*please fill out as much as you can below*
+
+# `<Country Code> - <Provider Name>`
+
+## Short description
+`<Brief description of the dataset, including key features such as spatial/temporal resolution, coverage, and data source.>`
+
+
+## 🗂️ Metadata
+
+- **📛 Dataset Name:** `<dataset-name>`
+- **⚙️ Status:** `<Active/Inactive>`
+- **⏰ Last Update:** `<YYYY-MM-DD>`
+- **📅 Update Frequency:** `<Frequency, e.g., Monthly, Periodic>`
+- **📏 Spatial Resolution:** `<e.g., 1 km x 1 km>`
+- **⏱️ Temporal Resolution (Timestep):** `<e.g., 5 minutes>`
+- **📐 Dimensions:** `<Height x Width in pixels>`
+- **🗄️ File Format (example filename):** `<e.g. HDF5 (RAD_NL25_RAC_RT_202411232055.h5)>`
+- **🗜️ Archive structure (example filename):** `<e.g. one file per timestep, daily TAR (RAD25_20241123.tar) or "N/A">`
+- **🔑 License:** `[<License Name>](<License URL>)`
+- **🌍 Country Coverage:** `<Country or Region>`
+- **🌐 Geographical Coverage:**
+  - **📍 North Bound Latitude:** `<Value>`
+  - **📍 East Bound Longitude:** `<Value>`
+  - **📍 South Bound Latitude:** `<Value>`
+  - **📍 West Bound Longitude:** `<Value>`
+- **📊 Variables (Units):** 
+  - `<Variable Name (units), for precipitation only "Rain rate (mm/h)", "Rainfall (mm)" or "Reflectivity (dBZ)" are accepted>`
+- **📬 Maintainer:** `<Maintainer Name/entity>`
+- **🕒 Temporal Coverage:**
+  - **⏳ Start Time:** `<YYYY-MM-DD>`
+  - **⏳ End Time:** `<YYYY-MM-DD or Unlimited>`
+- **🧭 Reference System Identifier:** `<Coordinate Reference System details (EPSG or proj string)>`
```

Comment:
I guess we could leave this out of the issue template since people will be adding this to the dataset anyway. But I am happy to keep it in too. These strings are just sometimes quite long..